userspace/libsinsp/event.h

```diff
@@ -238,7 +238,11 @@ class SINSP_PUBLIC sinsp_evt : public gen_event
 	*/
 	inline ppm_event_category get_category() const
 	{
-		return m_info->category;
+		// Bits >= EC_SYSCALL are used as "origin" flags;
```

Comment:
this solution has the problem of `EC_INTERNAL` that can be paired with other flags and so we cannot use `==` with the result of this function, an example is:

`PPME_CONTAINER_JSON_2_E` that has `EC_PROCESS | EC_INTERNAL`

Reply:
Solved in new commits :)